packages/client/components/TeamPrompt/TeamPromptTopBar.tsx

```diff
@@ -139,13 +141,25 @@ const TeamPromptTopBar = (props: Props) => {
 
   const onOpenWorkSidebar = () => {
     if (meeting.isRightDrawerOpen && meeting.showWorkSidebar && !meeting.localStageId) {
-      // If we're selecting a discussion that's already open, just close the drawer.
+      // If we're clicking on 'Your Work' when it's already open, just close the drawer.
+      SendClientSegmentEventMutation(atmosphere, 'Your Work Drawer Closed', {
+        teamId: meeting.teamId,
+        meetingId: meeting.id,
+        source: 'top bar'
+      })
+
       commitLocalUpdate(atmosphere, (store) => {
         const meetingProxy = store.get(meetingId)
         if (!meetingProxy) return
         meetingProxy.setValue(false, 'isRightDrawerOpen')
       })
     } else {
+      SendClientSegmentEventMutation(atmosphere, 'Your Work Drawer Opened', {
+        teamId: meeting.teamId,
+        meetingId: meeting.id,
+        source: 'top bar'
+      })
```

Comment:
-1 emit the metric after the local commit

Reply:
Fixed